docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+    habitat_sim.gfx.LightPositionModel.GLOBAL
+    habitat_sim.gfx.LightPositionModel.OBJECT
+
+Each LightSetup is registered in the simulator via a unique key. Two default lighting setups are pre-defined:
+
+
+- **habitat_sim.gfx.DEFAULT_LIGHTING_KEY**
+
+  8 white, GLOBAL lights at scene bounding box corners.
+
+- **habitat_sim.gfx.NO_LIGHT_KEY**
+
+  0 lights (for Flat shaded assets).
+
+
+Additional custom setups can be created and registered via the **simulator.set_light_setup** function:
+
+.. code:: python
+
+    sim.set_light_setup(new_light_setup, "my_custom_lighting_key")
+
+Any existing LightSetup can be queried with **simulator.get_light_setup**:
+
+.. code:: python
+
+    custom_light_setup = sim.set_light_setup(n"my_custom_lighting_key")
+
+
+`Working with Lights Example:`_
+===============================
+
+The example code below demonstrates the usage of default and custom LightSetups and is runnable via:
+
+.. code:: shell-session
+
+    $ python examples/tutorials/lighting_tutorial.py
+
+First, we import modules we will need, define some convenience functions, and initialize the Simulator, Agent, and objects.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [setup]
+    :end-before: # [/setup]
+
+`Scene Lighting`_
+=================
+
+By default, the scene will be shaded with no lights using the **habitat_sim.gfx.NO_LIGHT_KEY** LightSetup.
+This configuration is ideal for scene assets with illumination baked into textures (e.g. building scans such as MP3D).
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [default scene lighting]
+    :end-before: # [scene swap shader]
+
+.. image:: images/lighting-setups-images/0.jpg
+    :width: 45%
+
+To use a custom light setup for the scene, edit the **habitat_sim.SimulatorConfiguration.scene_light_setup**
+option when creating/reconfiguring your Simulator. Note that
+while you can modify the scene's LightSetup dynamically during runtime, you will need to
+reconfigure the simulator to switch the scene's LightSetup key. Also, due to asset loading specifics
+**you must close and re-initialize the Simulator to swap between Flat and Phong shading setups**.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [scene swap shader]
+    :end-before: # [/scene]
+
+
+
+.. |im1| image:: images/lighting-setups-images/1.jpg
+   :width: 100%
+
+.. |im2| image:: images/lighting-setups-images/2.jpg
+   :width: 100%
+
+.. |cp1| replace:: After loading with DEFAULT_LIGHTING_KEY.
+
+.. |cp2| replace:: After swapping to custom "my_scene_lighting".
+
++---------+-----------+
+|  |im1|  +   |im2|   +
+|  |cp1|  +   |cp2|   +
++---------+-----------+
+
+
+`Object Lighting`_
+==================
+
+By default, object assets are loaded with Phong shading compatability. This is not ideal for assets with illumination baked into textures.
```

Comment:
This right here is the beginning of a use-case. Could you expand this to say why someone might be introducing a new object asset (that it could be a robot mesh or a target object to navigate to or avoid)?

Reply:
I did another iteration on all the text. Should be more use-case driven now 👍  There will also be a complete tutorial specifically on objects (templates, instancing, manipulation, etc...). Once that is together we can cross link.